### ISSUE TITLE
Search box in embedded side panel is not highlighted when navigating from keyboard

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9028]
## Observed behavior
When a user uses a keyboard to navigate through the embedded side panel, when the navigation reaches the search box, there is no visual indication to the user that they are in the search box aside from the cursor blinking.

![2022-01-21 13 20 58](https://user-images.githubusercontent.com/13563002/150603362-473c589c-f08e-4858-b135-357c95661df0.gif)

## Errors and logs
No errors

## Expected behavior
There should be a highlight around the search box the way that there is in the `KTextbox` component. Similar to this:

## User-facing consequences
Unclear to users when they are able to type in the search box.

## Steps to reproduce
1. Log in as a learner on mobile and go to "Learn"
2. Use the keyboard to navigate onto the "Filter" button
3. Once inside, use the "tab" key to navigate to the search box. Note that there will be no highlighting around it.

## Context
Kolibri 0.15
Chrome

**Comment by himanshi18037** [comment]
Hi @sairina! This side Himanshi Mathur, I would like to work on this issue. I am able to reproduce the issue, but not able to locate the starter codebase in Kolibri repository, I am new towards the codebase, so can you guide a bit on it.

**Comment by sairina** [comment]
Hi @himanshi18037, that would be great. Thank you for taking this on. The component that the search box (`SearchBox`) is in is called `EmbeddedSidePanel`, which is in `FullScreenSidePanel`. The component that is currently handling how the focus is implemented is called `FocusTrap`.

1. There's a PR and a discussion in there about how `FocusTrap` was updated to be implemented [here](https://github.com/learningequality/kolibri/pull/9029), if it's helpful context.
2. The focus ring around the search box should look something like this
![Screen Shot 2022-03-25 at 10 09 43 AM](https://user-images.githubusercontent.com/13563002/160168762-d95d44b4-acd8-40be-bf23-9c0c0973693c.png)
4. Also, I've included a screenshot of how the components are rendered, if that's helpful. Here is where you would find those components in code:
- [`EmbeddedSidePanel`](https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue)
- [`FullScreenSidePanel`](https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/assets/src/views/FullScreenSidePanel/index.vue)
- [`FocusTrap`](https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/assets/src/views/FocusTrap.vue)

![Screen Shot 2022-03-25 at 10 00 07 AM](https://user-images.githubusercontent.com/13563002/160167090-3c584263-b8b6-4645-b01d-3bee76bc6e15.png)

**Comment by zhihanC** [comment]
Hi @himanshi18037, just want to check in if this issue is resolved yet. I'm a new contributor looking for a good first issue to work on. I'm happy to help with this one!

**Comment by rtibbles** [comment]
Fixed in #9806
